packages/orbit-components/src/LinkList/index.tsx

```diff
@@ -99,24 +15,34 @@ const LinkList = ({
   children,
   dataTest,
 }: Props) => (
-  <StyledLinkList
-    $indent={indent}
-    $direction={direction}
+  <ul
     data-test={dataTest}
-    $legacy={legacy}
-    $spacing={spacing}
+    className={cx(
+      "flex",
+      "w-full",
+      "m-0 p-0",
+      "list-none",
+      "text-normal",
+      indent && "ps-xxs",
```

Comment:
It does not. And the change made should be reverted. `ps-xxs` is more specific than `p-0`. We still need `p-0` to apply `padding: 0` to all other sides (except the inline-start one)

Reply:
Are you sure? If `padding-inline-start` is before `padding`, it gets overwritten. That was my concern. Can we ensure that `p-0` class is before `ps-xxs` in the final CSS? Maybe this is something Tailwind controls, I have no clue.

https://codi.link//PGRpdiBjbGFzcz0iYm94Ij7wn42FPC9kaXY+|LmJveCB7CiAgaGVpZ2h0OiAyMDBweDsKICB3aWR0aDogNDAwcHg7CiAgYmFja2dyb3VuZC1jb2xvcjogdG9tYXRvOwogIHBhZGRpbmctaW5saW5lLXN0YXJ0OiA1MHB4OwogIHBhZGRpbmc6IDEwcHg7Cn0=|

Am I doing something wrong?

Reply:
That's a very interesting question. I **_want to believe_** that Tailwind has that controlled in its compilation of the final css… That it will always put all-around-padding classes before, followed by two-sided-padding (X or Y) classes and, finally, one-side-padding classes… But that is a common sense belief I have. I found no documentation nor assurance that that happens…

Reply:
Ok! consider it resolved from my side.